charts/falco/templates/service.yaml

```diff
@@ -6,7 +6,10 @@ metadata:
   namespace: {{ include "falco.namespace" . }}
   labels:
     {{- include "falco.labels" . | nindent 4 }}
+    {{ toYaml .Values.metrics.service.labels | nindent 4 }}
```

Comment:
```suggestion
    {{- if gt (len .Values.metrics.service.labels) 0 }}
    {{ toYaml .Values.metrics.service.labels | nindent 4 }}
    {{- end }}
```

Reply:
We need to check that the labels' dict is not empty before rendering it. Otherwise, when no labels are specified we will get something like:
```yaml
  labels:
    helm.sh/chart: falco-4.8.3
    app.kubernetes.io/name: falco
    app.kubernetes.io/instance: release-name
    app.kubernetes.io/version: "0.38.2"
    app.kubernetes.io/managed-by: Helm
    
    {}
```